main/webapp/modules/core/MOD-INF/controller.js

```diff
@@ -278,6 +281,7 @@ function registerImporting() {
   IM.registerMimeType("text/xml", "text/xml");
   IM.registerMimeType("application/rdf+xml", "text/rdf/xml");
   IM.registerMimeType("application/ld+json", "text/rdf/ld+json");
+  IM.registerMimeType("application/atom+xml", "text/xml");
```

Comment:
I actually have some feeds aggregated using Mozilla FF addon's and would like this to be easier to just feed OpenRefine the URL's and then I can just quickly XML box select the Entry Elements in OpenRefine for sanitizing and reconciling within some domains.